source/renderer/app/containers/wallet/dialogs/WalletRestoreDialogContainer.js

```diff
@@ -36,7 +36,7 @@ export default class WalletRestoreContainer extends Component<Props> {
       restoreWalletCancelClose,
       restoreWalletChangeStep,
     } = actions.wallets;
-    const stepId = RESTORE_WALLET_STEPS[restoreWalletStep];
+    const stepId = RESTORE_WALLET_STEPS[restoreWalletStep || 0];
```

Comment:
@yakovkaravelov I'm not sure that this fallback to `0` is necessary because when the restoration process is started store will set the current step to 0. In this way, you will mutate / fake values and flow provided by the store.